lib/migration.js

```diff
@@ -423,7 +432,7 @@ function mixinMigration(MySQL, mysql) {
         fkSQL = self.getForeignKeySQL(model, newFks);
     }
     if (fkSQL && fkSQL.length) {
-      self.applySqlChanges(model, fkSQL, function(err, result) {
+      self.applySqlChanges(model, [fkSQL.join(', ')], function(err, result) {
```

Comment:
Can we do something like `[fkSQL.toString()]` instead? I assume you are to concatenate all the fkSQL's and join them with `,`. The `toString()` function on an array does that same thing in a more elegant way.